code/formfields/MollomField.php

```diff
@@ -64,6 +69,31 @@ public function getCaptcha() {
 
 		return null;
 	}
+	
+	/**
+	 * Set the captcha type. Mollom supports "image" and "audio" types.
+	 * 
+	 * @return void
+	 */
+	public function setCaptchaType($Type)
+	{
+		if (in_array($Type, array('image', 'audio')))
+		{
+			$this->captchaType = $Type;
+		}
+	}
+	
+	/**
+	 * Returns the captcha type that has been set. Default is "image".
+	 * 
+	 * @return string
+	 */
+	public function getCaptchaType()
+	{
+		return $this->captchaType;
+	}
+	
+	
```

Comment:
Reading through that link, it is mainly the braces on new lines that is the issue yeah? Will get another commit to fix that shortly.